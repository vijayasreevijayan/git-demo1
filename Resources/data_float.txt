1.234,13.567,19.89, 21.876, 34.67
3.56,4.67,9.78
8.54,32.41,76.8765