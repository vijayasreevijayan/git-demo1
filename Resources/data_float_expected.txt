data_float.txt
Line 1 average 18.2474 standard deviation 12.2161
Line 2 average 6.00333 standard deviation 3.31744
Line 3 average 39.2755 standard deviation 34.6817